dev-haskell/concurrent-supply/concurrent-supply-0.1.8-r1.ebuild

```diff
@@ -1,29 +1,37 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-# ebuild generated by hackport 0.6
+# ebuild generated by hackport 0.6.7.9999
 
 CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
 inherit haskell-cabal
 
 DESCRIPTION="A fast concurrent unique identifier supply with a pure API"
 HOMEPAGE="https://github.com/ekmett/concurrent-supply/"
-SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+SRC_URI="
+	https://hackage.haskell.org/package/${P}/${P}.tar.gz
+	https://hackage.haskell.org/package/${P}/revision/1.cabal -> ${PN}.cabal"
```

Comment:
The `distfiles` names have to be unique. Otherwise next bump will reuse (outdated) `${PN}.cabal` file for you on next bump like that. I suggest using something verbose but unambiguous, like `${PF}.cabal`.

The `DIST` entry above is a good test on a filename in `distfiles/`.

Same for other ebuilds.